debug_toolbar/settings.py

```diff
@@ -42,6 +42,7 @@
     "SQL_WARNING_THRESHOLD": 500,  # milliseconds
     "OBSERVE_REQUEST_CALLBACK": "debug_toolbar.toolbar.observe_request",
     "TOOLBAR_LANGUAGE": None,
+    "UDPATE_ON_AJAX": False,
```

Comment:
Typo (`UPDATE` not `UDPATE`)

Also, I think the setting should use `FETCH`, not `AJAX`, to reference the newer and preferred API.